wire-ios/Wire-iOS/Sources/UserInterface/E2EIdentity/OAuthUseCase.swift

```diff
@@ -42,7 +42,7 @@ public class OAuthUseCase: OAuthUseCaseInterface {
         logger.info("invoke authentication flow")
 
         guard let bundleID = Bundle.main.bundleIdentifier,
-              let redirectURI = URL(string: "\(bundleID):/oauth2redirect")
+              let redirectURI = URL(string: "wire://e2ei/oauth2redirect")
```

Comment:
did you test it on C builds? as wire is not a scheme, does that work?
you can change the WIRE_URL_SCHEME to check locally?

Reply:
yes, with `WIRE_URL_SCHEME = wire-c3-test`